app/tests/spec/views/confirm.js

```diff
@@ -363,7 +352,39 @@ define(function (require, exports, module) {
       it('triggers gmail tab opening', function () {
         sinon.spy(view.notifier, 'trigger');
         view._gmailTabOpened();
-        assert.isTrue(view.notifier.trigger.called);
+        assert.isTrue(TestHelpers.isEventLogged(metrics, 'confirm.openGmail.clicked'));
+      });
+    });
+
+    describe('openGmail feature', function () {
+      it('it is not visible in basic contexts', function () {
+        assert.notOk($('#open-gmail').length);
+      });
+
+      it('it is visible in desktop contexts', function () {
+        broker = new FxDesktopV3AuthenticationBroker({
```

Comment:
You don't actually need the FxDesktopV3AuthenticationBroker here, and it's actually a little brittle to do so. You could use a BaseBroker instance, add the capability, e.g., `broker.setCapability('openGmailButtonVisible', true);` then run the rest of the test.

I'd also rename the test to something like "is visible if the broker supports the `openGmailButtonVisible` capability"  or similar which is what we really want.